src/foam/nanos/controller/ApplicationController.js

```diff
@@ -481,26 +482,24 @@ foam.CLASS({
           // If the user has a personal theme, use that.
           this.theme = await this.user.personalTheme$find;
```

Comment:
Considering the potential that the user personalTheme cannot be found from the personalTheme$find (Missing in themeDAO), We could remove the else below and `return` if personalTheme$find is found to support further fall back.

Reply:
Updated.